P5/Source/Guidelines/en/CMC-ComputerMediatedCommunication.xml

```diff
@@ -0,0 +1,1302 @@
+          <post modality="written" type="tweet" generatedBy="human" synch="#tweetsbcrn18.t001"
+            xml:id="cmc_post_1043764753502486528" who="#u1" xml:lang="de">
+            <time generatedBy="system"> 12:31 </time> Heute mit super Unterstützung, wir grunzen,
+            wenn die Zeit vorbei ist. <ref type="hashtag"
+              target="https://twitter.com/hashtag/bcrn18?src=hash">#bcrn18</ref>
+            <ref type="hashtag" target="https://twitter.com/hashtag/wikidach?src=hash"
+              >#wikidach</ref> PS: Die beiden brauchen noch Namen. Hinweise dazu am Empfang abgeben!
+              <ref type="twitter-account" target="https://twitter.com/AndreLo79">@AndreLo79</ref>
+            <figure type="image">
+              <graphic url="https://pbs.twimg.com/media/DnwygdSW4AAoTUn.jpg:large"/>
+            </figure>
+          </post>
+          <post modality="written" generatedBy="unspecified" type="tweet" who="#u1"
+            synch="#tweetsbcrn18.t002" xml:id="cmc_post_1043769240136880128">
+            <ptr type="retweet" target="#cmc_post_1043767827927388160"/>
+          </post>
+          <post modality="written" generatedBy="human" type="tweet" who="#u3" synch="#tweetsbcrn18.t002"
+            xml:lang="de" xml:id="cmc_post_1043767827927388160">
+            <time generatedBy="system"> 12:43 </time>
+            <figure type="image" generatedBy="human">
+              <graphic url="https://pbs.twimg.com/media/Dnw1TRNXgAAKqlK.jpg:large"/>
+            </figure>
+          </post>
+        </egXML>
+      </p>
+      <p>Finally, in the following example of an IRC post, the status message that user <q>Interseb
+          has entered the room</q> was generated by the system, i.e. the chat environment. <egXML
+          xmlns="http://www.tei-c.org/ns/Examples" xml:id="ex.listPerson" xml:lang="de"
+          source="#BIB_DCK">
+          <post type="event" generatedBy="system" who="#SYSTEM" rend="color:navy">
+            <p><name type="nickname" ref="#A07">Interseb</name> betritt den Raum.</p>
+          </post>
+        </egXML>
+      </p>
+    </div>
+  </div>
+  <div xml:id="CMCmacrometa">
+    <head>CMC Macrostructure</head>
+    <p>In many CMC genres, posts come in a CMC document according to
+      which they are displayed, e.g. in a sequence or in threads;
+      posts within the document may be grouped. For example, in chat communication
+      such as WhatsApp, posts are part of <q>a chat</q> of one user with another user or among a
+      <!-- shouldn’t “a chat” above be <soCalled> like “logfile” ?? —Syd, 2024-01-31 -->
+      group of users. When an entire chat is saved, typically a <soCalled>logfile</soCalled> of the
+      chat is obtained from the CMC system and downloaded. Similarly, Wikipedia discussions come in
+      a <term>talk page</term>, which ultimately is a web page containing the user posts,
+      sub-structured in threads. Likewise, YouTube comments come in a webpage containing the YouTube
```

Comment:
Consider rephrasing as suggested above from "come in" to "occur in" or something similar perhaps?

Reply:
Tweaked wording in 759c2adba. Take a look and see if you think it is OK now.